packages/mdc-card/_mixins.scss

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/theme/mixins";
+
+//
+// Public
+//
+
+@mixin mdc-card-fill-color($color) {
+  @include mdc-theme-prop(background-color, $color);
+}
+
+@mixin mdc-card-corner-radius($radius) {
+  border-radius: $radius;
+}
+
+@mixin mdc-card-media-aspect-ratio($x, $y) {
+  &::before {
+    // This clever trick brought to you by: http://www.mademyday.de/css-height-equals-width-with-pure-css.html
+    margin-top: percentage($y / $x);
+  }
+}
+
+//
+// Private
+//
+
+@mixin mdc-card-box-sizing_ {
+  &,
+  &::before,
+  &::after {
+    box-sizing: border-box;
+  }
+}
+
+@mixin mdc-card-column-layout_($display: flex) {
```

Comment:
What is the point of the `display` parameter existing if we never set it and we pretty much assume it's flex given the presence of flex-direction, anyway?

Actually, do we even need this mixin? AFAICT these styles don't affect anything on `__media` and `__media-content` so they're only actually needed once, on `mdc-card`...

Reply:
Removed! Good catch!